ml-algorithms/src/test/java/org/opensearch/ml/engine/algorithms/remote/ConnectorUtilsTest.java

```diff
@@ -71,19 +71,19 @@ public void processInput_TextDocsInputDataSet_NoPreprocessFunction() {
         MLInput mlInput = MLInput.builder().algorithm(FunctionName.REMOTE).inputDataset(dataSet).build();
 
         ConnectorAction predictAction = ConnectorAction
-            .builder()
-            .actionType(PREDICT)
-            .method("POST")
-            .url("http://test.com/mock")
-            .requestBody("{\"input\": \"${parameters.input}\"}")
-            .build();
+                .builder()
```

Comment:
Added unnecessary tabs

Reply:
@ akolarkunnu as you can see from the stacktrace, the exception from software.amazon.awssdk.utils is not in our package, and when reaching to ConnectorUtil, the exception message is `host cannot be null`.  which is confusing to users. That's the intention to throw more meaningful message to customer, would you also try to mimic a invalid endpoint with a placeholder and see if the message make sense to you?

Reply:
I agree with that. You are already doing that in line no: 331. 
Why you need to catch and re-throw it again line no: 334.

Reply:
was trying to specify when the uri became null. let me update the message to be clear